### PR TITLE
JSON support

### PR DESCRIPTION
**Pull request by yflim**
Addresses #18.

##### Checklist
- [x] Implements an existing feature request. Make sure the feature request has been accepted for implementation before opening a PR.
- [x] Related issues linked using `fixes #number`
- [x] Integration tests added
- [x] Documentation added
- [ ] Architecture Decision Record added 
- [x] Formatting checked

### CODE REVIEW

**Comment by kordano** [comment]
Let me know when this is finished and all conflicts are resolved.

**Comment by yflim** [comment]
> Let me know when this is finished and all conflicts are resolved.

Done

File: src/clj/datahike_server/core.clj [hunk 1]
```diff
@@ -2,18 +2,10 @@
   (:gen-class)
   (:require [mount.core :as mount]
             [taoensso.timbre :as log]
-            [datahike-server.config :refer [config]]
-            [datahike-server.database]
```
Comment (kordano) [inline src/clj/datahike_server/core.clj:6]: both, `datahike-server.database` and `datahike-server.server` needs to be in here so the components can be mounted as you already mentioned in the tests.

**Review by kordano — CHANGES_REQUESTED** [review]
Very good job! Will be ready for merge once the small issue in the core is resolved.

**Review by kordano — APPROVED** [review]

**Pull request merged**